### DISCUSSION TITLE
Is scoped lifetime in V8 [ASP.NET DI] possible?

### DISCUSSION THREAD

**Discussion by rickifunk** [BrighterCommand/Brighter#1881] [Q&A] [answered]
Currently on the preview builds, its straight forward to change the lifetime of my handlers to scoped, how would this be done on V8? 

Currently i would either have to change my scoped services to be transient, or use one of the V9 preview builds (which i assume i should not do in production).

**Comment by preardon** [comment, marked as answer]
It is currently not possible in V8. V9 should be out soon and this is one of the new features.  

I am currently running v9 Preview in production (MSSQL Outbox, Azure Service Bus message transport) and it's running fine.  The only changes in Preview 6 are enhancements around the ASB transport and MySql outbox

**Reply by rickifunk** [reply]
Might just jump right into V9 then. I only have a couple of non critical events that needs to be handled for now. Thank you!
